cli/cliui/agent.go

```diff
@@ -381,6 +381,9 @@ func (d ConnDiags) Write(w io.Writer) {
 			_, _ = fmt.Fprintf(w, " - %s\n\n", msg)
 		}
 	}
+	if !d.PingP2P || d.Verbose || len(client) > 0 || len(agent) > 0 || len(general) > 1 {
+		_, _ = fmt.Fprintln(w, "For more information, see https://coder.com/docs/networking/troubleshooting")
```

Comment:
This link won't work if user doesn't have public internet access.  For a follow up PR, I think we accept a `CODER_DOCS_URL` setting on the server that it would be nice to pipe thru to here.

Reply:
There's a few places in the CLI where we're not respecting that deployment option - will plumb it in in a future PR.